src/services/nnas/index.ts

```diff
@@ -15,17 +16,39 @@ import oauth from '@/services/nnas/routes/oauth';
 import people from '@/services/nnas/routes/people';
 import provider from '@/services/nnas/routes/provider';
 import support from '@/services/nnas/routes/support';
+import settings from '@/services/nnas/routes/account-settings';
 
 // * Router to handle the subdomain restriction
 const nnas = express.Router();
 
+// Static routes for the user information app

```

Comment:
Nitpick. Make comment format match others

```suggestion
// * Static routes for the user information app
```